src/app/components/Images/Images.tsx

```diff
@@ -1,11 +1,11 @@
 import React from 'react';
-import styles from './Images.module.css';
 
 export type ImagesProps = {
-  image: 'Mino1' | 'Mino2' | 'Mino3' | 'Mino4';
+  image: string;
+  className: string | undefined;
 };
-export default function Images({ image }: ImagesProps): JSX.Element {
+export default function Images({ image, className }: ImagesProps): JSX.Element {
   return (
-    <img src={`src/assets/MockImages/${image}.png`} className={styles.Images} />
+    <img src={`src/assets/MockImages/${image}.png`} className={className} />
```

Comment:
This still doesn't solve the issue of having a fixed set of images, the source should be set where the component is implemented, or where the component which uses this component is implemented.